Author: Andreas Beckmann <anbe@debian.org>
Description: fix FTBFS with GCC 15

--- a/include/wngrind.h
+++ b/include/wngrind.h
@@ -154,7 +154,7 @@ extern char *PrintFileName(int);
 extern char *PrintPointer(Pointer);
 extern char *PrintSynonym(Synonym);
 extern char *NextFile();
-extern int filemode();
+extern int filemode(int);
 extern G_Synset CreateSynset(unsigned char, Synonym, Pointer,
 		    Framelist, char *, unsigned int, int, unsigned char);
 extern Pointer CreatePointer(Symbol, Symbol, unsigned char,
--- a/src/grind/create.c
+++ b/src/grind/create.c
@@ -108,7 +108,7 @@ Framelist CreateFramelist(int frameval)
     return(f);
 }
 
-static unsigned hash();
+static unsigned hash(char *s);
 
 Symbol CreateSymbol(char *s)
 {
--- a/src/grind/grind.c
+++ b/src/grind/grind.c
@@ -10,7 +10,7 @@
 
 static char *Id = "$Id: grind.c,v 1.33 2005/01/31 20:03:36 wn Rel $";
 
-static void badarg(), GetSenseCounts(), Stats(), SymStats();
+static void badarg(char *), GetSenseCounts(), Stats(), SymStats();
 static int grind();
 int verifyflag = 0, statsflag = 0, distribflag = 0;
 int senseindex = 0;
--- a/src/grind/wnlex.l
+++ b/src/grind/wnlex.l
@@ -25,11 +25,9 @@ int yyinhead=0;
 int commentline=0;
 int pcount=0;
 
-static void fixdigit();
-static void i_c_err();
-static int outmess();
-void yyerror(), yywarn();
-char *strclone();
+static void fixdigit(char *);
+static void i_c_err(char);
+void yyerror(char *), yywarn(char *);
 
 int curfilenum(void);
 int curfilemode(void);
@@ -328,7 +326,7 @@ yywrap()
     
     if (pcount) {
 	fprintf(logfile, "%s: Comment started on line %d left open\n",
-	       filelist[curfilenum()], commentline);
+	       filelist[curfilenum()].fname, commentline);
 	errcount++;
     }
     return 1 ;
--- a/src/grind/wnparse.y
+++ b/src/grind/wnparse.y
@@ -28,7 +28,6 @@ static Synonym AddSynonym(Synonym, Synon
 static Pointer AddPointer(Pointer, Pointer);
 static Synonym InsertSynonym(Synonym, Synonym);
 static void AddSynset(G_Synset);
-static void AddToSynList();
 static void AddToSynList(G_Synset);
 static void MakeSyngroup(Pointer, Framelist);
 static Pointer DoPointer(Symbol, int);
