Author: Andreas Beckmann <anbe@debian.org>
Description: fix FTBFS with GCC 14

--- a/src/grind/grind.c
+++ b/src/grind/grind.c
@@ -19,7 +19,9 @@ int nominalizations = 0;
 int synsetkeys = 0;
 int extraflag = 0;
 
-void main(argc,argv)
+void yyparse();
+
+int main(argc,argv)
 char **argv;
 int argc;
 {
--- a/src/grind/wnlex.l
+++ b/src/grind/wnlex.l
@@ -31,6 +31,9 @@ static int outmess();
 void yyerror(), yywarn();
 char *strclone();
 
+int curfilenum(void);
+int curfilemode(void);
+
 /*
  * Now some weird stuff to change the behavior of Lex so that it
  * remembers which state to start in even after returning a token.
@@ -319,6 +322,7 @@ char *s;
 	errcount++;
 }
 
+int
 yywrap()
 {
     
--- a/src/grind/wnparse.y
+++ b/src/grind/wnparse.y
@@ -37,6 +37,13 @@ static int UniquePointer(Pointer, Pointe
 static int UniqueSynSense(Synonym, int);
 static int UniqueFanSynonym(Synonym, int, int);
 static char *UseLabel(char *);
+
+int yylex(void);
+void yyerror(char *);
+void yywarn(char *);
+
+int curfilenum(void);
+int curfilemode(void);
 %}
 
 %start wordnet
@@ -379,7 +386,7 @@ defn:	{ $$ = NULL; }
 
         ;
 
-key:    { $$ = NULL; }
+key:    { $$ = 0; }
 
         | SYNSETKEY { $$ = $1; }
 
